sampler.go

```diff
@@ -384,34 +398,57 @@ func (s *httpSamplingManager) GetSamplingStrategy(serviceName string) (*sampling
 	return &out, nil
 }
 
+// SetMaxOperations sets the maximum number of operations the sampler will keep track of.
+func SetMaxOperations(maxOperations int) func(*RemotelyControlledSampler) {
```

Comment:
a couple of stylistic preferences:
 * because we're not using a different package for sampling, these functions pollute the official `jaeger` package, which is not necessary. You can see how this is solved with tracer options, by defining option methods on a factory struct `TracerOption` instead of as public functions. I would suggest the same approach here, e.g. with a `SamplerOption` factory struct
 * because these methods are "options", it is customary not to use `Set` prefix